src/reducers/language.js

```diff
@@ -0,0 +1,19 @@
+export default function(state='norwegian', action) {
+  switch (action.type) {
+    case 'SET_LANGUAGE':
+      return isLanguageValid(action.payload) ? action.payload : null;
+  }
+  return state;
+}
+
+function isLanguageValid(language) {
+  const languages = [
+    'norwegian',
+    'norwegian-nynorsk',
+    'swedish',
+    'danish',
+    'english'
+  ];
```

Comment:
Burde disse språkene hentes fra en config-fil, eller er det greit at de er hardkodet her? Hva blir enklest for de som skal vedlikeholdet dette senere?